pkg/plugin/sdk/authz/authz.proto

```
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package gameap.plugin.sdk.authz;
```

Buf (1.71.0)

[error] Files with package "gameap.plugin.sdk.authz" must be within a directory "gameap/plugin/sdk/authz" relative to root but were in directory "pkg/plugin/sdk/authz".

(PACKAGE_DIRECTORY_MATCH)